README.md

```diff
@@ -59,25 +59,25 @@ let g:lightline.component_type = {
 
 " Add the components to the lightline:
 let g:lightline.active = {
-  \   left': [[ 'lsp_info', 'lsp_hints', 'lsp_errors', 'lsp_warnings', 'lsp_ok' ], [ 'lsp_status' ]]
+  \   'left': [[ 'lsp_info', 'lsp_hints', 'lsp_errors', 'lsp_warnings', 'lsp_ok' ], [ 'lsp_status' ]]
   \ }
 ```
 
 ## Configuration
 
-- `g:lightline#lsp#indicator_warnings`
+- `let g:lightline#lsp#indicator_warnings`
   The indicator to use when there are warnings. Default is `•`.
 
-- `g:lightline#lsp#indicator_errors`
+- `let g:lightline#lsp#indicator_errors`
   The indicator to use when there are errors. Default is `×`.
 
-- `g:lightline#lsp#indicator_info`
+- `let g:lightline#lsp#indicator_info`
   The indicator to use when there are information messages. Default is `~`.
 
-- `g:lightline#lsp#indicator_hints`
+- `let g:lightline#lsp#indicator_hints`
   The indicator to use when there are hints. Default is `>`.
 
-- `g:lightline#lsp#indicator_ok`
+- `let g:lightline#lsp#indicator_ok`
```

Comment:
Could you revert these changes? I don't think should be in here.

Reply:
Done